app/views/rating_questions/edit.html.erb

```diff
@@ -0,0 +1,20 @@
+
+<div class="form"><%= form_with model: @rating_question, url: rating_question_path,local: true do |f| %>
+  <%= f.label :title %>
+  <%= f.text_field :title %>
+  <%= f.label :tag %>
+  <%= f.text_field :tag %>
+  <%= f.submit %>
+  <% end %>
+  <%= link_to 'Back', rating_questions_path %>
+  <div class="flash-notice"><%= flash[:notice] %>
+  </div>
+  <div class="single">
+    <div class='questions'>
+      <% if @rating_question %>
```

Comment:
I am not sure why this exists here. We should take a look at it together.

Reply:
interesting, it was breaking previously without it...